examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<blockquote>Model training finishes without error. It means that we can use the&nbsp;<code>device</code>&nbsp;and the&nbsp;<code>microbatch</code>&nbsp;together.</blockquote><p>strange conclusion in tutorial. </p><p> </p><blockquote>last model.</blockquote><p>Use same notation for all models. (Model with two GPUs and microbatch)</p><p> </p><p>Small conclustions should be after each sections, not only in the end. Last section should sum up the whole tutorial.</p><p> </p><blockquote>Furthermore, when we added microbatch to the second model (made the last model), we increased model training time.</blockquote><p>Why is happend?</p><p> </p><blockquote>&nbsp;(see cells with inline magic&nbsp;<code>%%time</code>).</blockquote><p>use profiling instead (<a href="https://github.com/analysiscenter/batchflow/blob/master/examples/tutorials/08_profiling.ipynb" target="_blank">https://github.com/analysiscenter/batchflow/blob/master/examples/tutorials/08_profiling.ipynb</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>